src/smart-components/OpenChannelSettings/components/OperatorUI/OperatorsModal.tsx

```diff
@@ -0,0 +1,117 @@
+import React, {
+  ReactElement,
+  useContext,
+  useEffect,
+  useState,
+} from 'react'
+
+import Modal from '../../../../ui/Modal';
+import UserListItem from '../../../../ui/UserListItem';
+import IconButton from '../../../../ui/IconButton';
+import Icon, { IconTypes, IconColors } from '../../../../ui/Icon';
+import ContextMenu, { MenuItem, MenuItems } from '../../../../ui/ContextMenu';
+
+import useSendbirdStateContext from '../../../../hooks/useSendbirdStateContext';
+import { useOpenChannelSettingsContext } from '../../context/OpenChannelSettingsProvider';
+import { LocalizationContext } from '../../../../lib/LocalizationContext';
+
+interface Props { onCancel?(): void }
+
+export default function OperatorListModal({ onCancel }: Props): ReactElement {
+  const [users, setUsers] = useState([]);
+  const [operatorQuery, setOperatorQuery] = useState(null);
+
+  const { channel } = useOpenChannelSettingsContext();
+  const state = useSendbirdStateContext();
+  const currentUser = state?.config?.userId;
+  const { stringSet } = useContext(LocalizationContext);
+
+  useEffect(() => {
+    const participantListQuery = channel?.createParticipantListQuery({
+      limit: 20,
+    });
+    participantListQuery.next().then((participants) => {
+      setUsers(participants);
+    });
+    setOperatorQuery(participantListQuery);
+  }, [])
+  return (
+    <div>
+      <Modal
+        hideFooter
+        titleText="All users"
```

Comment:
Will be good to localize `All users` (in other files also)

Reply:
Let's do that with string set task